docs/guides/transfer_learning.md

```diff
@@ -118,9 +119,9 @@ params = variables['params']
 Since `params` are currently random, the pretrained parameters from `vision_model_vars` have to be transfered to the `params` structure at the appropriate location. This can be done by unfreezing `params`, updating the `backbone` parameters, and freezing the `params` again:
 
 ```{code-cell} ipython3
-from flax.core.frozen_dict import freeze
+from flax.core.frozen_dict import unfreeze, freeze
 
-params = params.unfreeze()
+params = unfreeze(params)
 params['backbone'] = vision_model_vars['params']
 params = freeze(params)
```

Comment:
If this change lands I think we can just remove `freeze` and `unfreeze` altogether here.